tools/slang-unit-test/unit-test-coverage-metal-runtime.cpp

```
@@ -0,0 +1,327 @@
+    slang::IComponentType* components[] = {module, entryPoint};
+    ComPtr<slang::IComponentType> program;
+    SLANG_CHECK_ABORT(
+        session->createCompositeComponentType(components, 2, program.writeRef(), nullptr) ==
+        SLANG_OK);
+
+    ComPtr<slang::IComponentType> linked;
+    diagnostics.setNull();
+    SLANG_CHECK_ABORT(program->link(linked.writeRef(), diagnostics.writeRef()) == SLANG_OK);
+    diagnoseIfNeeded(diagnostics, "link");
+
+    diagnostics.setNull();
```

Cppcheck (2.21.0)

[style] The function 'Swap' is never used.

(unusedFunction)